CONTRIBUTING.md

```diff
@@ -0,0 +1,36 @@
+# Contributing
+
+The dance tool and tests run on the host; macOS, Linux, and Windows are expected to work.
+Databases under test (FerretDB and MongoDB) may be running on the host or inside Docker; Docker Compose configuration is provided for convenience, but not required.
+In particular, the FerretDB development cycle (fix-compile-run-dance) is faster with it running on the host as it does not involve Docker images building or PostgreSQL restarts.
+
+## Running tests
+
+```sh
+make dance DB=ferretdb TEST=ferret
```

Comment:
what parameters `DB` and `TEST` mean? 

here if assume `DB` - to be compared to and `TEST` - what is being tested, it is confusing that they both have `ferret` in word. I thought, DB=mongo and TEST=ferret.

Reply:
That's explained below